### ISSUE TITLE
Error when running on ARM platform

### ISSUE THREAD

**Issue by lucaspal** [aptabase/aptabase#65]
I plan to self host the service on a arm based VPS. The `aptabase_app` container spins up but in an error state

When executing `docker compose up` I get the following warning:

```sh
 ! aptabase The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested               0.0s 
```

The container logs are the following:

```sh
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
exec /usr/bin/dotnet: exec format error
```

While the warning is not necessarily a blocking issue (but it makes sense to add the platform as supported once this is fixed), the error makes the app unusable.

EDIT: I am trying to run it on a `aarch64` based machine

**Comment by goenning** [comment]
Yeah we only have x64 images at the moment, so that error is expected.
I'm not very familiar with multi-arch images though in dotnet though

**Comment by jcpsantiago** [comment]
Looks like the SudoVanilla images are gone. Does anyone know another source of arm images? I'm assuming this is not a priority for the Aptabase team at the moment and likely something the community will take over.

**Comment by AlexProgrammerDE** [comment]
Hi, I'm also looking for a way to run aptabase on an arm machine. (Oracle cloud free tier)
Here is an example of how to target multiple platforms in a docker build: https://github.com/AlexProgrammerDE/SoulFire/blob/0eb57227ea9e8fe549de3f217a996d3ba922b246/.github/workflows/docker-deploy.yml#L39
It runs a virtual arm envrionment on the github hosted runner using the QEMU builder installed previously in that script. I'll look right now into whether this can be integrated with the current aptabase CI using a PR.

**Comment by jcpsantiago** [comment]
@AlexProgrammerDE maybe this is relevant https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/ no more QEMU needed :)

**Comment by AlexProgrammerDE** [comment]
I'm not so sure how well it works with dotnet, but alright...

**Comment by ninjadev64** [comment]
What I do on my Raspberry Pi is clone this repo locally, then specify the `build` key with the path to the cloned repo for my compose.yaml container.
